lib/rexml/document.rb

```diff
@@ -443,6 +443,17 @@ def record_entity_expansion
       end
     end
 
+    attr_accessor :namespaces_cache
+
+    def enable_namespaces_cache
```

Comment:
How about removing `_namespaces`?
I don't want to add more similar `enable_XXX_cache` internal APIs in the future...